### PR TITLE
chore: bump to godot 4.3 and gdext 0.1.3

### PR DESCRIPTION
**Pull request by leanmendoza**
- set rustc 1.79 (at least 1.78 required by gdext 0.1.3)
- bump gdext to production release v0.1.3
- bump godot from 4.2.1 to 4.3

# Changes
- (animation controller) add TimeSeek, time by paremeters is no longer supported in 4.3
- if a scene throw an error, it doesn't crash, instead, try to show the error. if it's spaming, it stops showing it => fix genesis plaza random error

### CODE REVIEW

**Comment by kuruk-mm** [comment]
Doors in Genesis Plaza were panicking when you approach to those. Add a checker for cast to f32, but there is still a missing "set", because the "get" returns a null.

File: build-android-apk.sh [hunk 1]
```diff
@@ -46,7 +51,25 @@ cd ${EXPLORER_PATH}/godot/
...
+# Try executing the command
+if ! $COMMAND; then
+    echo "First attempt failed, retrying in 5 seconds..."
+    sleep 5 # Wait for 5 seconds before retrying
+
+    # Retry executing the command
+    if ! $COMMAND; then
+        echo "Second attempt failed."
+    else
+        echo "Second attempt succeeded."
```
Comment (leanmendoza) [inline build-android-apk.sh:67]: hackerman-meme-1

Comment (kuruk-mm) [reply build-android-apk.sh:67]: haha did you try with a sleep before the first attempt?

**Comment by kuruk-mm** [comment]
Android Test:
- Profile image is not being processed
- Goerli plaza, advanced-avatar-swap (76,-6) and avatar-swap (75,-7) the animations doesn't plays and always looks forward

**Review by kuruk-mm — APPROVED** [review]

File: .github/workflows/android-build.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 on:
   push:
     branches:
-    - main
+    # - main
```
Comment (kuruk-mm) [inline .github/workflows/android-build.yml:4]:
```suggestion
    - main
```

before merging

**Review by kuruk-mm — APPROVED** [review]

**Pull request merged**